tests/test_main.py

```diff
@@ -88,6 +88,22 @@ def test_batch_mode_table(executor):
     assert result.exit_code == 0
     assert expected in result.output
 
+@dbtest
+def test_batch_mode_csv(executor):
+    run(executor, '''create table test(a text, b text)''')
+    run(executor, '''insert into test (a, b) values('abc', 'def'), ('ghi', 'jkl')''')
+
+    sql = 'select * from test;'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, args=CLI_ARGS + ['--csv'], input=sql)
+
+    expected = 'a,b\nabc,def\nghi,jkl\n'
+    result_output = result.output.replace('\x00', '') # python 3
```

Comment:
@tsroten You were right. I removed the `truncate` calls and the `\x00` took vacation. 😄 
Thanks, man!